### PR TITLE
Add tests for Goal

### PR DESCRIPTION
**Pull request by lue97**

### CODE REVIEW

**Comment by ivantjh** [comment]
How did u manage to solve the error ah?

File: src/test/java/seedu/address/model/person/GoalTest.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.parser.exceptions.ParseException;
+
+public class GoalTest {
+
+    private static final List<String> VALID_WEEK_INPUTS = Arrays.stream(new String[]{
```
Comment (ivantjh) [inline src/test/java/seedu/address/model/person/GoalTest.java]: `Arrays.asList(new String[] {"w", "week", "weekly"})` shorter line to convert to List

File: src/test/java/seedu/address/model/person/GoalTest.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+    @Test
+    public void goal_equals_failure() {
+        Object o = new Object();
+        assertNotEquals(new Goal(), o);
+    }
+
+    @Test
+    public void goal_emptyGoal_success() {
+        assertEquals(new Goal(Goal.Frequency.NONE), new Goal());
```
Comment (ivantjh) [inline src/test/java/seedu/address/model/person/GoalTest.java]: Use static import for the whole file, will be more readable

File: src/test/java/seedu/address/model/person/GoalTest.java [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    }
+
+    @Test
+    public void goal_getFrequency_success() {
+        assertEquals(Goal.Frequency.NONE, new Goal(Goal.Frequency.NONE).getFrequency());
```
Comment (ivantjh) [inline src/test/java/seedu/address/model/person/GoalTest.java]: You should test something other than `NONE` since the default value is `NONE`

**Review by ivantjh — APPROVED** [review]
lgtm!

File: src/test/java/seedu/address/model/person/EmailTest.java [hunk 4]
```diff
@@ -1,61 +0,0 @@
-package seedu.address.model.person;
-
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static seedu.address.testutil.Assert.assertThrows;
-
-import org.junit.jupiter.api.Test;
-
-public class EmailTest {
```
Comment (ivantjh) [inline src/test/java/seedu/address/model/person/EmailTest.java]: Why are we removing this? This seems valid.

**Comment by lue97** [comment]
> How did u manage to solve the error ah?

I was working on a out of date branch and was making use of `EventBuilder` which was removed in another PR.

File: src/test/java/seedu/address/model/person/EmailTest.java (hunk 4 above)
Comment (lue97) [reply src/test/java/seedu/address/model/person/EmailTest.java]: My bad this was unintended thanks for catching it.

**Comment by ivantjh** [comment]
Ohh ok

**Pull request merged**